.github/workflows/workflow-maintenance.yml

```
@@ -1,4 +1,14 @@
+on:
+  workflow_dispatch:
+
+permissions:
+  contents: read
+
+concurrency:
+  group: ${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
 
 on:
```

YAMLlint (1.37.1)

[error] duplication of key "on" in mapping

(key-duplicates)